src/coalescenceml/integrations/tensorflow/step/tensorflow_trainstep.py

```diff
@@ -4,23 +4,31 @@
 from tensorflow.keras.models import Sequential
 from tensorflow.keras.layers import Dense
 from keras.callbacks import EarlyStopping, ModelCheckpoint
+from coalescenceml.step import BaseStep
+from coalescenceml.step import BaseStepConfig
+
+class TFClassifierTrainConfig(BaseStepConfig):
+    layers: typing.List
+    hyperparams: typing.Dict[str, typing.Any] = {}
```

Comment:
Can we document what hyperparameters are even available? For sklearn, it was hard b/c each model was different but here there is only 1 model.

Reply:
Modified this to be more specific.